doc/doc_02_cap_01.rst

```diff
@@ -254,7 +254,7 @@ Esempi di SLI sono i seguenti:
 
 -   dimensione massima di ogni richiesta accettata. Le richieste più grandi possono essere rifiutate;
 
--   latenza al 90° percentile. Utilizzata per calcolare la responsività;
+-   latenza al 90º percentile. Utilizzata per calcolare la responsività;

```

Comment:
cambia il carattere?

Reply:
Il simbolo di [grado](https://it.wikipedia.org/wiki/Grado_(simbolo)) (°) è stato sostituito con l'[indicatore ordinale](https://it.wikipedia.org/wiki/Indicatore_ordinale) (º).